tsconfig.json

```diff
@@ -1,10 +1,10 @@
 {
 	"compilerOptions": {
 		"module": "commonjs",
-		"target": "ES2022",
+		"target": "ESNext",
```

Comment:
oh dear why. Which feature necessitates this and is there good support in Electron?

Reply:
Very good catch, I don't recall why I modified this. I have reverted it to ES2022, which is also what the webclient now depends on following this PR: https://github.com/spcl/dace-webclient/pull/181

Note that we _would_ in theory be fine with `ESNext`, as VSCode's version of Electron uses a version of the V8 engine which supports quite a few features that are not even in the most recent ECMA standard yet, but this is definitely the safer option.